src/cljam/io/csi.clj

```diff
@@ -66,15 +81,12 @@
     (->> #(let [bin (lsb/read-int rdr)
                 loffset (lsb/read-long rdr)
                 chunks (read-chunks! rdr)]
-            (hash-map
-             :bin bin
-             :loffset loffset
-             :chunks chunks))
+            {:bin (long bin) :loffset loffset, :chunks chunks})
```

Comment:
Using same style in maps might be better.
```suggestion
            {:bin (long bin), :loffset loffset, :chunks chunks})
```

Reply:
Thanks! Fixed in 89ae85e